test/functional/transport/transport.transactions.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+process.env.SILENT = true;
```

Comment:
We should set this in the test command, not here - we might want to add a command that runs these tests with `process.env.SILENT = false`. (E.g. on Jenkins?)